Tests/ML/test_download_upload.py

```diff
@@ -188,7 +188,19 @@ def run() -> Any:
 def test_mount_in_azureml1(test_output_dirs: OutputFolderForTests) -> None:
     """
     Test cases when MLRunner.mount_or_download_dataset runs inside AzureML.
+    First files from 'data.csv' are copied to the mounted folder and then lightning container is mounted and tested.
     """
+    df = pandas.read_csv(str(full_ml_test_data_path(DATASET_CSV_FILE_NAME)), usecols=['filePath'])
+
+    path = test_output_dirs.root_dir / "mounted"
+    path.mkdir(exist_ok=True)
+
+    train_and_test_data_path = path / "train_and_test_data"
+    train_and_test_data_path.mkdir(exist_ok=True)
+
+    for filePath in set(df['filePath'].values):
+        shutil.copy(full_ml_test_data_path() / filePath, path / filePath)
+
```

Comment:
as discussed, it would be simpler to make a change in _test_mount_for_lightning_container. There, the dataset.csv file is already copied to both "mounted" and "downloaded" folders.

Reply:
Implemented in _test_mount_for_lightning_container.